coreneuron/mechanism/eion.cpp

```diff
@@ -277,14 +259,16 @@ void nrn_cur_ion(NrnThread* nt, Memb_list* ml, int type) {
     double* pd;
     Datum* ppd;
     (void) nt; /* unused */
-#if defined(_OPENACC)
-    int stream_id = nt->stream_id;
-#endif
     /*printf("ion_cur %s\n", memb_func[type].sym->name);*/
     int _cntml_padded = ml->_nodecount_padded;
     pd = ml->data;
     ppd = ml->pdata;
-    _PRAGMA_FOR_CUR_ACC_LOOP_
+    nrn_pragma_acc(parallel loop present(
```

Comment:
Nitpick: indentation is a bit messed up? I would say that something like:
```
        pd [0:_cntml_padded * 5],
        nrn_ion_global_map[0:nrn_ion_global_map_size] [0:ion_global_map_member_size])
        if (nt->compute_gpu) async(nt->stream_id))
```
would be more understandable. I don't know if this is possible to do easily

Reply:
Improved